vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -224,7 +217,8 @@ def parse_selector_parts(entity)
       result
     end
 
-    def parse_container_definition(container_def, pod_id)
+    def parse_container_definition(container_def, pod)
+      pod_id = pod.metadata.uid
```

Comment:
If you modify the code above according to my comments then you don't need this change here.